parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -302,6 +302,9 @@ cleanUpContainerImages() {
         docker rmi $(docker images --format '{{.Repository}}:{{.Tag}}' | grep 'kube-svc-redirect') &
         docker rmi $(docker images --format '{{.Repository}}:{{.Tag}}' | grep 'nginx') &
     fi
+
+    # TZL Financials: https://github.com/Azure/aks-engine/issues/1625
+    docker rmi registry:2 &
```

Comment:
looks like you're removing this image for all clusters, wasn't there an Azure Stack specific cleanup at some point that we could reuse?